FlowControlHomework/FlowControlHomework/FlowControl.cs

```diff
@@ -347,7 +352,7 @@ public void Loops()
                 if (i == 15) Assert.IsTrue(true);
             }
 
-           int max2 = 17;
+            int max2 = 17;
```

Comment:
I have a mistake here. The test doesn't actually fail when the comment says `if you reach here test will fail`.
But there's a corrected version of this problem on the new FlowControl homework, so I'll accept this one.

Reply:
Thank you